src/styles/main.scss

```diff
@@ -0,0 +1,65 @@
+@import "./extends";
+@import "./variables";
+
+body {
+  margin: 0;
+  background-color: #2d5074;
+}
+
+.stopwatch {
+  height: 100vh;
+}
+
+.clock {
+  width: $clock-width;
+  height: $clock-height;
+
+  @extend %centring;
```

Comment:
your `.clock` is a BEM block. Do not give it positioning.
you already have a wrapper class `.stopwatch`. simply give your clock one more class, something like `stopwatch__clock`. this way your clock will have two classes: `clock` and `stopwatch__clock`. then move all the positioning to `stopwatch__clock`